toolchain/check/check_unit.cpp

```diff
@@ -325,6 +334,37 @@ auto CheckUnit::ImportOtherPackages(SemIR::TypeId namespace_type_id) -> void {
   }
 }
 
+auto CheckUnit::ImportCppPackages() -> void {
+  for (const auto& import : unit_and_imports_->cpp_imports) {
+    llvm::StringRef cpp_file_path =
+        unit_and_imports_->unit->value_stores->string_literal_values().Get(
+            import.library_id);
+
+    auto file = fs_->openFileForRead(cpp_file_path);
+    if (!file) {
+      CARBON_DIAGNOSTIC(CppInteropFileNotFound, Error,
```

Comment:
```suggestion
      CARBON_DIAGNOSTIC(CppInteropCantReadFile, Error,
```
The text and the name say different things, they could be more similar?

Reply:
Done.